### DISCUSSION TITLE
database system in recovery mode

### DISCUSSION THREAD

**Discussion by Ophrys78** [teslamate-org/teslamate#3333] [Q&A]
Hello,

Everything seems to work well, except when I try to load the "Visited" view. I get:
"db query error: pq: the database system is in recovery mode" in the corner of the graph.

Docker’s logs give me:
```
teslamate_1  | 2023-09-04 22:05:36.746 [error] Postgrex.Protocol (#PID<0.2427.0>) disconnected: ** (DBConnection.ConnectionError) tcp recv (idle): closed
teslamate_1  | 2023-09-04 22:05:37.875 [error] Postgrex.Protocol (#PID<0.2434.0>) failed to connect: ** (Postgrex.Error) FATAL 57P03 (cannot_connect_now) the database system is in recovery mode
```
(ten times each line)

It is fairly recent (maybe related to the growth of the database?).
Any idea? Thanks!

**Comment by nilsmau** [comment]
@Ophrys78 how did you solve it? just wait?

**Comment by Ophrys78** [comment]
@nilsmau it has been solved by an update some weeks / months ago. I couldn't say which one…
